internal-resources/git-workflow-and-contributor-guide.adoc

```diff
@@ -4,135 +4,226 @@
+* https://ohmyz.sh/
+* https://fig.io/
 
-IMPORTANT: If you are suddenly prompted for a password when you attempt to access GitLab, it is possible that you have overwritten your SSH public key. Once you have set up your SSH public key and authenticated to GitLab, be careful not to overwrite it. If you generate a new public SSH key for another product and overwrite this one, you will lose your authentication to GitLab! If this happens, you must add your new SSH public key to GitLab.
 
-=== Fork the Upstream Repository
+=== Installing a text editor
 
-Fork the `stonesoup-documentation` upstream repository to create a copy under your own GitLab ID. This allows you to work on multiple features and push changes to branches in your own GitLab instance so you don't have to worry about losing work. When you are ready, you can request the changes to be merged back into the upstream repository.
+Download a text editor. Our team generally uses https://code.visualstudio.com/[Visual Studio (VS) Code].
 
-. Open a browser and navigate to the https://gitlab.cee.redhat.com/red-hat-stone-soup/stonesoup-documentation.
-. Select the https://gitlab.cee.redhat.com/red-hat-stone-soup/stonesoup-documentation/-/forks/new[Fork] icon.
+=== Installing Antora
+{Product_Name} documentation uses Antora as a publishing tool. To contribute to the project, install https://docs.antora.org/antora/latest/install/install-antora/[Antora].
 
-+
-image::images/fork.png[alt=Fork]
+==== Installing Antora on a Mac
+The process to install Antora on a Mac is slightly different. 
 
-. Choose your user account for the location of the forked repository. This creates your own copy of the repository under your own GitLab ID.
+Prerequisites:
 
-=== Clone Your Forked Repository
+You must have https://brew.sh/[Homebrew] installed. 
 
-Clone your forked repository to bring your GitLab repository files to your local machine. Your forked repository is now the `origin` repository for your local files.
+Procedure: 
 
-. Open a terminal and navigate to the directory where you want to create the new repository folder.
-. Type the following command.
-+
-[source,bash,options="nowrap"]
-----
-$ git@gitlab.cee.redhat.com:<username>/stonesoup-documentation.git
-----
-. Navigate to the newly created `stonesoup-documentation` folder.
-+
-[source,bash,options="nowrap"]
-----
-$ cd stonesoup-documentation/
-----
+* Download https://go.dev/[golang] and https://nodejs.org/en[nodejs].
+* In the CLI, run `brew install make`.
+* Run `sudo rm /usr/local/bin/make`. 
+* Run `npm install`
+* Run `npm run build` 
 
-For more information about http://doc.gitlab.com/ee/workflow/forking_workflow.html[forking] and http://doc.gitlab.com/ce/gitlab-basics/command-line-commands.html#clone-your-project[cloning], consult the official http://doc.gitlab.com/ce/[Gitlab documentation].
+NOTE: The first time you do the preceding steps, the CLI might take several minutes to load. 
```

Comment:
Optional: the note might be better placed before the procedure so users know to expect wait times. However, since the procedure is pretty short, I think it's likely users will see and read the note before starting the procedure even though the note is at the end.